### PR TITLE
analytics (fixes #55)

### LINKED ISSUE #55: Add basic telemetry
We can start simple. We can measure what are the most used components

### PR DESCRIPTION
**Pull request by ngokevin**
- [x] Components - addComponent - componentName
- [x] Components - changeProperty - componentName
- [x] Components - removeComponent - componentName
- [x] <s>Components - copyComponent - componentName</s> _needs dependencies fixed_
- [x] <s>Components - resetComponent - componentName</s> _removed_
- [x] Components - collapse
- [x] Components - addMixin - componentName
- [x] Components - removeMixin - componentName
- [x] ToolBar - selectHelper - helperName
- [x] ToolBar  - toggleLocal - boolean
- [x] Scenegraph- selectEntity
- [x] Scenegraph - searchEntity
- [x] Scenegraph - navigateWithKeyboard
- [x] ExportMenu - exportHTML
- [x] ExportMenu - copyHTML
- [x] CreateMenu - createEntity - entityType
- [x] Viewport.pan **need to hack EditorControls to differentiate between these three**
- [x] Viewport.zoom
- [x] Viewport.rotate
- [x] Viewport - transformEntity - transformMode
- [x] Viewport - toggleEditor - boolean
- [x] Viewport - selectEntity

### CODE REVIEW

File: src/lib/viewport/index.js [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
 var Events = require('../Events');
-var getNumber = require('../utils').getNumber;
+
+const gaTrackTransformEntity = debounce(transformMode => {
+  ga('send', 'event', 'Viewport', 'transformEntity', transformMode);
+}, 100);
+
+const gaTrackChangeEditorCamera = debounce(() => {
+  ga('send', 'event', 'Viewport', 'changeEditorCamera');
+}, 100);
```
Comment (fernandojsg) [inline src/lib/viewport/index.js]: When dragging around the objects, will `100`ms beenough for debounce?

Comment (ngokevin) [inline src/lib/viewport/index.js]: Right, maybe 3-5 seconds would be better.

**Comment by fernandojsg** [comment]
Looks good to me. Should we merge already to start tracking? or leave it until the first "oficial" release to start tracking data?

**Comment by ngokevin** [comment]
Just merge and then later I can have an ENV flag to switch to the proper GA ID depending on development vs. production. Currently, it's hooked to my Mozilla Google account. But later, we want it in the A-Frame account.

@cvan Earlier you mentioned Potatolytics, but the comments are gone now Does that need to be done? This can track `localhost` for now, but if we need to `iframe`, we can layer that on.

**Comment by fernandojsg** [comment]
Ok, cool

**Pull request merged**